src/Global/Auth/authContext.js

```diff
@@ -0,0 +1,5 @@
+import { createContext } from "react";
+
+const UserContext = createContext("User");
```

Comment:
Default value means the object you are storing in the context. e.g.

```js

const UserContext = createContext({
    username: "",
    email: "",
    address: {
        street: "",
        city: "",
    }
});
```

Whatever it is according to your context.